packages/eas-cli/src/submissions/ios/AppProduce.ts

```diff
@@ -33,16 +33,9 @@ export async function ensureAppStoreConnectAppExistsAsync(
 
   const { bundleIdentifier, appName, language } = ctx.commandFlags;
 
-  let resolvedBundleId: string;
-  try {
-    const description = getProjectConfigDescription(ctx.projectDir, projectConfig);
-    resolvedBundleId =
-      bundleIdentifier ??
-      (await getBundleIdentifier(ctx.projectDir, exp, {
-        displayAutoconfigMessage: false,
-        configDescription: description,
-      }));
-  } catch (e) {
+  const resolvedBundleId =
+    bundleIdentifier ?? (await getAppIdentifierAsync(ctx.projectDir, Platform.iOS));
+  if (!resolvedBundleId) {
     throw new Error(
       `Please define "expo.ios.bundleIdentifier" in your app config or provide it using --bundle-identifier param.
 Learn more here: https://expo.fyi/bundle-identifier`
```

Comment:
In the future we should make this more like a controlled, failing warning message like https://github.com/expo/expo-cli/blob/2ac0ea6834727f346d5c6e55fc458b73d01d5254/packages/expo-cli/src/commands/init.ts#L132-L142 and reserve errors for unexpected functionality.